deliverables/README.md

```diff
@@ -12,16 +15,37 @@ Sign Up for Cloud9 and create a basic Rails project.
 ####Required
 
 What is an "interpreted" language?<br>
+In an interpreted language, the code we write is translated and run through an interpreter which performs the user's desired actions.
+
 What is IRB?<br>
+IRB stands for interactive Ruby. It interactively executes whatever Ruby code we write one line at a time.
+
 What are Objects?<br>
+Every thing in Ruby is an object. Objects are classified by the characteristics which make up a class of objects. In addition to an objects defining characteristics, you can perform functions/methods on objects.
+
 What are Methods?<br>
+Methods, or functions. are a set of instructions bound to an object and have access to its instance variables. You call methods on objects.
+
 What are Classes?<br>
+A class is the blueprint for which objects are created. Classes classify objects and each class of object has certain characteristics and functions associated to them.
+
 What are Blocks?<br>
+Blocks in Ruby are sets of instructions, placed inside curly braces, which can take arguments and return values and are invoked using a yield statement. They are syntax literals for Proc objects.
+
 What is an Array?<br>
+An array is a list of stored information. The list in an array has order. 
+
 What is an Iterator?<br>
+An iterator specifies the way a method might go through a collection of objects. Examples of iterators are "each" which returns each value of a collection one by one and "collect" which returns the entire collection of objects.
+
 What are hashes?<br>
+A hash, also known as a dictionary, stores pairs of related information. Each data set has a reusable lable to call on it: a key and a value, so as opposed to an array, order in a hash doesn't matter which is why it's like a dictionary that has a vocab word (key) and a definition (value).
+
 What is a library?<br>
+Libraries are sets of pre-written functions by a third party that can be downloaded for use and then accessed by calling on them in your own Ruby code.
+
```

Comment:
Yeah so added functionality for you to work off of, which abstracts away a lot of low level or logic into usable methods. Keep in mind, you too can create your own reusable library.